cip.go

```diff
@@ -83,11 +83,14 @@ func main() {
 		"version",
 		false,
 		"print version")
-	genImagesPtr := flag.Bool(
-		"gen-images",
-		false,
-		"generating all images in destination"+
-			" repository instead of promoting images")
+	snapshotPtr := flag.String(
+		"snapshot",
+		"",
+		"read all images in a repository and print to stdout")
+	snapshotSvcAccPtr := flag.String(
+		"snapshot-service-account",
```

Comment:
Maybe it should just be a separate command?  (In future?)

Reply:
As in like https://github.com/spf13/cobra (which supports multi-command option handling, a la kubectl)? YES. This is a no-brainer, but it is low on the priority list.

Reply:
Cool, SGTM

Reply:
I made an issue for this: #61